staging/src/k8s.io/node-api/pkg/apis/node/v1alpha1/types.go

```diff
@@ -0,0 +1,63 @@
+/*
```

Comment:
can you add an owners file similar to https://github.com/kubernetes/kubernetes/pull/70453/files#diff-14f4c6ed84723241de20108c87ae2c7d to the `node-api/pkg/apis` dir, swapping out a node reviewers alias in the reviewers section?

xref https://github.com/kubernetes/kubernetes/pull/70453

Reply:
Done.